app/views/jobseekers/subscriptions/index.html.haml

```diff
@@ -3,8 +3,31 @@
 = render partial: 'account_header'
 
 .govuk-width-container
-  .govuk-grid-row
+  .govuk-grid-row{ class: 'govuk-!-margin-left-0 govuk-!-margin-right-0' }
     .govuk-grid-column-full-width
       %h1.govuk-heading-l= t(".page_title")
 
+      %table.govuk-table
+        %thead.govuk-table__head
+          %th.govuk-table__header{ class: 'govuk-!-width-two-thirds' }= t(".heading_criteria")
+          %th.govuk-table__header= t(".heading_frequency")
+          %th.govuk-table__header.govuk-table__header--numeric= t(".heading_actions")
+        %tr.govuk-table__row
+          %td.govuk-table__cell
+            %ul.govuk-list
+              %li
+                %span{ class: 'govuk-!-font-weight-bold' }= 'key'
+                = 'value'
+              %li
+                %span{ class: 'govuk-!-font-weight-bold' }= 'key'
+                = 'value'
+
+          %td.govuk-table__cell= 'Daily, at approx 3pm'
+          %td.govuk-table__cell.govuk-table__header--numeric
+            %ul.govuk-list
+              %li
+                %a.govuk-link{ href: '/' }= 'Manage alert'
```

Comment:
We tend to use `link_to` helpers (and soon `govuk_link_to` helpers)

Reply:
ok i see that as being added a subsequent PR though